houndigrade/cli.py

```diff
@@ -0,0 +1,215 @@
+        '{}'.format(partition),
+        '{}'.format(inspect_path)
+    ],
+        check=True,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        universal_newlines=True
+    )
+    yield mount_result
+    subprocess.run(['umount', '{}'.format(inspect_path)])
+
+
+def report_results(results):
+    """
+    Places the results on a queue.
+
+    Args:
+        results (dict): The results of the finished inspection.
+
+    """
+    queue_name = os.getenv('RABBITMQ_QUEUE_NAME')
+
+    exchange = Exchange(os.getenv('RABBITMQ_EXCHANGE_NAME'), durable=True)
+    queue = Queue(queue_name, exchange=exchange, routing_key=queue_name)
+
+    with Connection(os.getenv('RABBITMQ_URL')) as conn:
+        producer = conn.Producer(serializer='json')
+        producer.publish(
+            results,
+            exchange=exchange,
+            routing_key=queue_name,
+            declare=[queue]
+        )
+
+
+def check_release_files(partition, results):
+    """
+    Check os release files to see if they indicate RHEL.
+
+    Args:
+        partition (str): The partition we are currently checking.
+        results (list): Part of the results dict to which we should be
+        writing our results.
+
+    """
+    release_file_paths = find_release_files()
+
+    if not release_file_paths:
+        click.echo('No release files found on {}'.format(partition))
+        results.append({
+            'rhel_found': False,
+            'status': 'No release files found on {}'.format(partition)
+        })
+    else:
+        for release_file_path in release_file_paths:
+            rhel_found, contents = check_file(release_file_path)
+
+            if rhel_found:
+                click.echo('RHEL found on: {}'.format(partition))
+            else:
+                click.echo('RHEL not found on: {}'.format(partition))
+
+            results.append({
+                'rhel_found': rhel_found,
+                'release_file': release_file_path,
+                'release_file_contents': contents
+            })
+
+
+def find_release_files():
+    """
+    Look for potential release files on the mounted volume.
+
+    Returns (list): List of file system paths that much the pattern.
+
+    """
+    return glob.glob('{}/etc/*-release'.format(INSPECT_PATH))
+
+
+def get_partitions(drive):
+    """
+    Look for partitions belonging to an attached volume.
+
+    Args:
+        drive (str): The drive to check.
+
+    Returns (list): List of file system paths that much the pattern.
+
+    """
+    return glob.glob('{}*[0-9]'.format(drive))
+
+
+def check_file(file_path):
+    """
+    Check the release file to see if it indicated a RHEL OS.
+
+    Args:
+        file_path (str): Path of the file to check.
+
+    Returns:
+        boolean: Whether we think this file is from RHEL.
+        file (str): The contents of the release file to support our decision.
+
+    """
+    try:
+        with open(file_path) as f:
+            file = f.read()
```

Comment:
Did you forget to rename `file` or decide not to?

Reply:
😅